php_phongo.c

```diff
@@ -606,6 +649,10 @@ static int phongo_execute_parse_options(mongoc_client_t* client, int server_id,
 				if (!process_read_preference(*driver_option, mongoc_opts, zreadPreference, client, server_id TSRMLS_CC)) {
 					return false;
 				}
+			} else if ((!strcmp(ZSTR_VAL(string_key), "session", zsession, client))) {
```

Comment:
strcmp here, but strcasecmp below?

Reply:
Knowing that we were not going to allow case-insensitive options with [PHPC-1057](https://jira.mongodb.org/browse/PHPC-1057), I opted not to use `strcasecmp()` when I implemented this. That said, this code no longer exists after the refactoring in #705, since we know use the `php_array` functions to access it.